localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```
@@ -18,3 +19,187 @@
+ 
+ innovation_hexer:0 "Hexers"
+ innovation_hexer_desc:0 "The extremely cruel and nefarious troll Hexers are extremely skilled at manipulating the chemical processes in friendly warriors to increase their fighting abilities. A few of them know ancient rituals to temporarily break or even take over the wills of enemy soldiers."
+
+ innovation_kodo:0 "Kodo Taming"
+ innovation_kodo_desc:0 "Tauren legends abound regarding the great horned kodos of Kalimdor’s Barrens. Some myths tell of rare kodos that are bound to the spirits of the sky and storm, thus allowing the colossal beasts to harness the powers of lightning itself. While kodos are generally considered gentle by nature, they are fierce opponents when threatened and have occasionally been known to devour their enemies whole. Our tamers have successfully learned how to communicate and tame these grand creatures."
+ 
+ innovation_marauder:0 "Marauders"
+ innovation_marauder_desc:0 "Like most of their kin, Marauders are savagery and brutality incarnate. Marauders are fast, aggressive and keen to show no mercy. Dawning the hides of their enemies, Marauders are not known for taking prisoners."
+    
+ innovation_magi:0 "Magi"
+ innovation_magi_desc:0 "Ogre Magi are skilled spellcasting warriors. The two-headed ogre spellcasters are typically larger and more intelligent than their one-headed brethren. They often enjoy an elevated position among the ogre clans."
+    
+ innovation_mountain_giant:0 "Mountain Giants"
+ innovation_mountain_giant_desc:0 "Mountain Giants are slow melee units that can take an incredible amount of damage. These creatures, created by Titans from solid rock ages ago, strive to protect the natural environment from anything that could harm it. Their titanic origins give them a naturally high resistance to physical and magical attacks."
+ 
+ innovation_naga_ground:0 "Tidal Terrors"
+ innovation_naga_ground_desc:0 "With our unique ground armaments, we have produced regiments like the Myrmidons and the Sirens."
+ 
+ innovation_naga_taming:0 "Songs of Sorrow"
+ innovation_naga_taming_desc:0 "We have tamed the Couatl and enslaved the Sea Giants, and now we have incorporated them into our army."
+ 
+ innovation_savage:0 "Savages"
+ innovation_savage_desc:0 "Savages are fanatic and barbaric warriors of our clan. They are strong, stubborn, and always looking to spill some blood."
+ 
+ innovation_silithid:0 "Silithid"
+ innovation_silithid_desc:0 "Silithids are tough and sophisticated insectoid creatures. The chitin plates of their bone armor are intertwined in such a way that it makes it incredibly hard. Lumped in huge numbers, the silithids are devouring swarms. Stopping at nothing but the demise of other life on Azeroth."
```

Too many its, gets confusing. I also changed the last sentence a bit, as it was a dependent clause.
```suggestion
 innovation_silithid_desc:0 "Silithids are tough and sophisticated insectoid creatures. The chitin plates of their bone armor are intertwined in such a way that it makes their armor incredibly hard. Lumped in huge numbers, the silithids are devouring swarms. They stop for nothing but the demise of other life on Azeroth."
```